File: news-front/src/main/java/io/md/news/model/ArticleView.java
package io.md.news.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ArticleView {
    private String image;
    private String title;
    private List<String> paragraphs;
}

File: news-front/src/main/js/layout/article-view.js
const React = require('react');
const ReactDOM = require('react-dom');

export default class ArticleView extends React.Component {
  createParagraphs_(body) {
    var paragraphs = [];
    for(let i = 0; i < body.paragraphs.length; i++) {
      var component = <p key={i}>{body.paragraphs[i]}</p>
      paragraphs.push(component);
    }
    return paragraphs;
  };

  render() {
    var body = window.DATA.body;
    var paragraphs = this.createParagraphs_(body);
    return (
      <div className="card-panel article-view">
        <div className="article-view-img">
          <img src={body.image}></img>
        </div>
        <h4>{body.title}</h4>
        {paragraphs}
      </div>
    );
  }
}